llama_stack/providers/inline/agents/meta_reference/agent_instance.py

```diff
@@ -911,7 +911,7 @@ async def load_data_from_url(url: str) -> str:
 
 
 async def get_raw_document_text(document: Document) -> str:
-    if not document.mime_type.startswith("text/"):
+    if not (document.mime_type.startswith("text/") or document.mime_type == "application/yaml"):
```

Comment:
`text/yaml` is considered [deprecated](https://www.rfc-editor.org/rfc/rfc9512.html#name-media-type-application-yaml) 

> Deprecated alias names for this type: ...`text/yaml`...

Therefore `application/yaml` is the only official MIME Type for YAML documents.

Reply:
I believe that is another issue, how to handle the deprecated mime type.
Would you have any preference if I should include/allow more types (json/xml) in this PR?

Reply:
yes we can followup on this.